packages/api-utility/src/position.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Only necessary in Electron and OpenFin as browser does not know about
+ * more than 1 screen.
+ */
+export const getPositionFromDisplay = (displayId: string, position: ssf.Position): Promise<ssf.Position> => {
```

Comment:
Maybe this should export a `Position` or `Display` class that has a static method? It just means that the namespace for api-utility will be better organised as it grows (i.e. not be full of loads of functions)